src/main/java/org/folio/rest/impl/JobNumberAPI.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.rest.impl;
+
+import static org.folio.rest.util.ResponseUtils.buildErrorResponse;
+import static org.folio.rest.util.ResponseUtils.buildOkResponse;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.jaxrs.model.SequenceNumber;
+import org.folio.rest.jaxrs.resource.FinanceStorageJobNumber;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.utils.TenantTool;
+
+public class JobNumberAPI implements FinanceStorageJobNumber {
+
+  private static final Logger log = LogManager.getLogger(JobNumberAPI.class);
+  private static final String JOB_NUMBER_QUERY = "SELECT nextval('job_number')";
+
+  @Override
+  public void getFinanceStorageJobNumber(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    log.debug("Trying to get job number");
+    vertxContext.runOnContext((Void v) -> {
```

Comment:
runOnContext is not needed here, so can simplify

Reply:
simplfieid